src/components/Create/components/pages/utils/rules/inputAlreadyExistsRule.ts

```diff
@@ -0,0 +1,18 @@
+import { t } from '@lingui/macro'
+import { RuleObject } from 'antd/lib/form'
+
+/**
+ * Rule to determine if the input already matches other `inputs`.
+ */
+export const inputAlreadyExistsRule = (props: { inputs: string[] }) => ({
+  validator: (rule: RuleObject, value: unknown) => {
+    if (value === undefined) return Promise.resolve()
+    if (typeof value !== 'string')
+      return Promise.reject(t`Invalid type - contact Juicebox Support`)
```

Comment:
Eh 😂 . Is there any other guidance we can give here?

Reply:
Realistically this shouldn't happen - if it does, we got a bug in our code